src/NuGet.Services.AzureSearch/Analysis/PackageIdCustomTokenizer.cs

```diff
@@ -3,15 +3,15 @@
 namespace NuGet.Services.AzureSearch
 {
     /// <summary>
-    /// Splits tokens that on a set of symbols.
-    /// For example, "Foo.Bar" becomes "Foo" and "Bar".
+    /// Splits tokens that on a set of symbols and whitespace.
+    /// For example, "Foo.Bar Baz" becomes "Foo", "Bar", and "Baz".
     /// </summary>
     public static class PackageIdCustomTokenizer
     {
         public const string Name = "nuget_package_id_tokenizer";
 
         public static readonly PatternTokenizer Instance = new PatternTokenizer(
             Name,
-            @"[.\-_,;:'*#!~+()\[\]{}]");
+            @"[.\-_,;:'*#!~+()\[\]{}\s]");
```

Comment:
BTW, Its fine to have here, but I think its only applicable to the DescriptionAnalyzer, since the ID wouldn't actually have spaces in it. In case you think we want it to be applied to specific analyzers.

Reply:
Yup, that's correct. I kept this as one so that we wouldn't need to mirror future changes to `PackageIdCustomTokenizer` to this second tokenizer. That being said, we can reassess if requirements change.